File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task167_PERPLEXITY_llama-3-sonar-large-32k-chat.java
public class Main {
    public static void main(String[] args) {
        System.out.printf("%.2f%n", cubeDiagonal(8));  // 3.46
        System.out.printf("%.2f%n", cubeDiagonal(343));  // 12.12
        System.out.printf("%.2f%n", cubeDiagonal(1157.625));  // 18.19
    }
    public static double cubeDiagonal(double volume) {
        return Math.round(Math.sqrt(3) * Math.pow(volume, 1. / 3) * 100.) / 100.;
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task167_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <cmath>
using namespace std;
double cubeDiagonal(double volume) {
    return round(sqrt(3) * pow(volume, 1. / 3) * 100.0) / 100.0;
}
int main() {
    cout << cubeDiagonal(8) << endl;
    cout << cubeDiagonal(343) << endl;
    cout << cubeDiagonal(1157.625) << endl;
    return 0;
}